token_update_transaction_hip540_e2e_test.go

```diff
@@ -203,8 +202,6 @@ func TestIntegrationTokenUpdateTransactionUpdateLowerPrivilegeKeysWithInvalidKey
 	require.NoError(t, err)
 	_, err = resp.SetValidateStatus(true).GetReceipt(env.Client)
 	require.ErrorContains(t, err, "INVALID_SIGNATURE")
-	fmt.Println(resp)
```

Comment:
I've seen lots of prints when running GO tests maybe we should eliminate those as well?

Reply:
yeah i will take some time to remove these in the future.